java/cdk-environment/src/main/resources/cloud-init.sh

```diff
@@ -53,9 +53,9 @@ sudo apt install -y docker-ce docker-ce-cli containerd.io binfmt-support qemu-us
 usermod -aG docker "$LOGIN_USER"
 
 # Install Sleeper CLI
-curl "https://raw.githubusercontent.com/$FORK/$REPOSITORY/$BRANCH/scripts/cli/install.sh" -o "$LOGIN_HOME/sleeper-install.sh"
+curl "https://raw.githubusercontent.com/$FORK/$REPOSITORY/develop/scripts/cli/install.sh" -o "$LOGIN_HOME/sleeper-install.sh"
```

Comment:
It looks like there's no guarantee this will exist on the fork, and the script won't pull from there anyway. Maybe we shouldn't use the FORK and REPOSITORY variables here?